src/WorldStorage/NBTChunkSerializer.cpp

```diff
@@ -610,6 +610,13 @@ void cNBTChunkSerializer::AddMonsterEntity(cMonster * a_Monster)
 				m_Writer.AddByte("IsConverting", (((const cZombie *)a_Monster)->IsConverting() ? 1 : 0));
 				break;
 			}
+			case mtRabbit:
+			{
+				const cRabbit & Rabbit = *((const cRabbit *) a_Monster);
```

Comment:
Then change the surrounding code. We are moving over from c style casts, its just there are a lot of them to onvert.